modules/xz/5.4.5/MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "xz",
+    version = "5.4.5",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "platforms", version = "0.0.8")
```

Comment:
@meteorcloudy Should we use here always the latest and greatest (in this case "0.0.8" - or should we really go for the minimal version that just works (maybe "0.0.4")?

Reply:
It's fine to use whatever version, just keep in mind that there is no need to keep it updated unless you actually need new features. If you do know the minimum supported version, using that is probably slightly better in general, but for something like `platforms`, some other rulesets will very likely force a higher version anyway.

Reply:
Okay perfect! Will leave at 0.0.8 for now as I don't think it's an issue